libs/features/footer-promo.js

```diff
@@ -0,0 +1,41 @@
+import { createTag } from '../utils/utils.js';
+
+async function getFooterPromoByTag(contentRoot) {
+  const NAME = 'Name';
+  const FOOTER_PROMO_LINK = 'Footer Promo Link';
+  const taxonomyUrl = `${contentRoot}/taxonomy.json`;
+  const tagsOnPage = [...document.head.querySelectorAll('meta[property="article:tag"]')].map((el) => el.content);
+
+  if (!tagsOnPage.length) return false;
```

Comment:
Since the expected return value is `primaryTag[FOOTER_PROMO_LINK];` and not a Boolean (true) - I would just `return` (undefined) or `null` if you really want to indicate nothing was returned.